mitmproxy/proxy/protocol/websocket.py

```diff
@@ -111,8 +165,9 @@ def _handle_connection_closed(self, event, source_conn, other_conn, is_server):
 
         print(self.connections[other_conn])
         self.connections[other_conn].close(event.code, event.reason)
+        other_conn.send(self.connections[other_conn].bytes_to_send())
+        source_conn.send(self.connections[source_conn].bytes_to_send())
```

Comment:
Immediately sending close frame to other end because after we call `self.connections[other_conn].close(event.code, event.reason)` it won't generate any more events.